src/pages/studyView/virtualStudy/VirtualStudy.tsx

```diff
@@ -0,0 +1,248 @@
+                                                className="btn btn-default share-cohort"
+                                                type="button"
+                                                disabled={this.buttonsDisabled}
+                                                onClick={(event) => { this.sharing = true; }}>
+                                                {this.sharing ? <i className="fa fa-spinner fa-spin" aria-hidden="true"></i> : "Share"}
+                                            </button>}
+                                        </div>
+                                    </div>
+                                    <textarea
+                                        className="form-control"
+                                        rows={10}
+                                        placeholder="Virtual study description (Optional)"
+                                        value={this.description}
+                                        onChange={event => this.description = event.currentTarget.value}
+                                    />
+                                </div>
+                                <span style={{ 'display': 'block', 'font-weight': 'bold' }}>This virtual study was derived from:</span>
+                                <div className={styles.studiesSummaryInfo}>
+                                    {
+                                        this.props.studyWithSamples.map(study => <StudySummaryRecord {...study} />)
+                                    }
+                                </div>
+                            </div>
+                        </Then>
+                        <Else>
+                            <div className={classnames(styles.virtualStudyResult)}>
+                                <a
+                                    target='_blank'
+                                    href={`${this.virtualStudyUrl}`}
+                                    style={{ 'margin-right': '10px' }}>
+                                    {this.virtualStudyUrl}
+                                </a>
+                                <div className="btn-group btn-group-xs">
+
+                                    <DefaultTooltip
+                                        overlay={
+                                            <If condition={this.copied}>
+                                                <Then>
+                                                    <span className="alert-success">Copied!</span>
+                                                </Then>
+                                                <Else>
+                                                    <span>Copy</span>
+                                                </Else>
+                                            </If>}
+                                        placement="top"
+                                        mouseLeaveDelay={0}
+                                        mouseEnterDelay={0.5}
```

Comment:
We shouldn't add delay. The content of the tooltip may actually lose the synchronization from the actions. The click even can actually triggered before the tooltip showing up.